Shaders/ssrefr_pass/ssrefr_pass.frag.glsl

```diff
@@ -102,27 +100,27 @@ void main() {
 	n = normalize(n);
 
     vec3 viewNormal = V3 * n;
-    vec3 viewPos = normalize(getPosView(viewRay, d, cameraProj));
-    vec3 refracted = refract(viewPos, viewNormal, 1.0 / ior);
+    vec3 viewPos = getPosView(viewRay, d, cameraProj);
+    vec3 refracted = refract(-viewPos, viewNormal, ior);
     hitCoord = viewPos;
 
-#ifdef _CPostprocess
-    vec3 dir = refracted * (1.0 - rand(texCoord) * PPComp10.y * roughness) * 2.0;
-#else
+//#ifdef _CPostprocess TODO
+///    vec3 dir = refracted * (1.0 - rand(texCoord) * PPComp10.y * roughness) * 2.0;
+//#else
     vec3 dir = refracted * (1.0 - rand(texCoord) * ss_refractionJitter * roughness) * 2.0;
-#endif
+//#endif
```

Comment:
What's the reason for all the `_CPostprocess` related TODOs here and in some other places in this file?

Reply:
The file is a modifies copy from ssr_pass.
Since I haven't implemented the C postprocess I commented out the related code.